### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by mentality2**
* Added ability for authentication to GET
* Updated readme for installing and running the app on a local test server
* Make basic calls and dump data on a html template page

### CODE REVIEW

File: KOA/clanwars/api.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class HabiticaApi(habitica.api.Habitica):
+    def __init__(self, user_id, api_token):
+        headers = {
+            AUTH_URL: "https://habitica.com",
+            #default.AUTH_HEADER_CLIENT: user_id,
+            #default.AUTH_HEADER_TOKEN: api_token
+
+            AUTH_HEADER_CLIENT: "54333f60-395f-42d0-8461-f5b59080498e",
+            AUTH_HEADER_TOKEN: "baa8235b-15f7-4a6b-a5a4-c71a2dc96a12"
```
Comment (ninichat) [inline KOA/clanwars/api.py]: Please remove these from the history.
See: https://help.github.com/articles/removing-sensitive-data-from-a-repository/

Comment (ninichat) [reply KOA/clanwars/api.py]: Also, email habitica about changing your api token if not already done :)

**Review by ninichat — COMMENTED** [review]
Might be urgent.

File: KOA/clanwars/api.py (hunk 1 above)
Comment (mentality2) [reply KOA/clanwars/api.py]: dammit i thought i got rid of that. it's only been up for a few hours tho hopefully not too bad

File: KOA/clanwars/apps.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 from django.apps import AppConfig
-
+import json
```
Comment (ninichat) [inline KOA/clanwars/apps.py]: Is this import useful?

Comment (mentality2) [reply KOA/clanwars/apps.py]: yes it allows for use of the json() function to parse data as json so it can be read

**Pull request closed without merging**